.travis.yml

```diff
@@ -23,3 +23,6 @@ matrix:
       gemfile: gemfiles/rails_5.2.gemfile
     - rvm: jruby-9.1.16.0
       gemfile: gemfiles/rails_5.2.gemfile
+script:
+- bundle exec rake
```

Comment:
What does this line achieve?

Also, since this no longer supports ruby 2.1, can we cleanup some of the excludes in the matrix?